docs/xgcpolicy.md

```diff
@@ -42,32 +42,31 @@ Controls the behavior of the garbage collector by specifying different garbage c
 | [`optthruput`](#optthruput)                                                  |         |
 | [`nogc`](#nogc)                                                              |         |
 
-
-Specify the garbage collection policy that you want the OpenJ9 VM to use:
+For a detailed description of the policies, when to use them, and how they work, see [Garbage Collection policies](gc.md). The following GC policies are available:
 
 ### `gencon`
 
         -Xgcpolicy:gencon
 
-: The generational concurrent policy (default) uses a concurrent mark phase combined with generational garbage collection to help minimize the time that is spent in any garbage collection pause. This policy is particularly useful for applications with many short-lived objects, such as transactional applications. Pause times can be significantly shorter than with the `optthruput` policy, while still producing good throughput. Heap fragmentation is also reduced.
+: The generational concurrent policy (default) requires a heap that is divided into two main areas (*nursery* and *tenure*) to manage two generation groups (*new* and *older*). The policy uses a global GC cycle of concurrent *mark-sweep* operations, optionally followed by *compact* operations. The policy also uses a partial GC cycle to run *scavenge* operations on the *nursery* area. The partial cycle helps reduce the frequency and duration of the global GC cycle. Note that *scavenge* is a *stop-the-world* operation, unless `-Xgcpolicy:gencon` is specified with the [`-Xgc:concurrentScavenge`](xgc.md#concurrentscavenge) option (supported only on IBM Z&reg; systems).
```

Comment:
`(supported only on IBM Z&reg; systems)` This is not accurate any more. IBM Z provides Hardware-based support for Concurrent Scavenge(z14 and higher). There in number of platforms with Software-based support   Please update correspondently (see `#### Concurrent Scavenge` section in gc.md)

Reply:
Oh yes .... the SW based update must have missed fixing the command ref. Will sort.